daemon/main.go

```diff
@@ -138,6 +138,11 @@ func init() {
 						Name:        "keep-config",
 						Usage:       "When restoring state, keeps containers' configuration in place",
 					},
+					cli.BoolFlag{
+						Destination: &config.KVStoreIPv4Registration,
```

Comment:
For now, this could be done by the node itself by storing some metadata such as the hostid or the UUID of the fstab together with the registered prefix and verify that this is the same